cores/esp8266/core_esp8266_postmortem.cpp

```diff
@@ -222,7 +233,24 @@ static void uart1_write_char_d(char c) {
 }
 
 static void raise_exception() {
-    __asm__ __volatile__ ("syscall");
+#if 0
+
```

Comment:
We should drop these lines completely.  Git will preserve the old info if needed.